lib/parsers/index.ts

```diff
@@ -244,13 +244,25 @@ export function getTargetFrameworksFromProjectFile(manifestFile) {
     return targetFrameworksResult;
   }
   const propertyList = projectPropertyGroup
-    .find((propertyGroup) => _.has(propertyGroup, 'TargetFramework'));
-
-  if (!propertyList) {
+    .find((propertyGroup) => {
+        return _.has(propertyGroup, 'TargetFramework')
+        || _.has(propertyGroup, 'TargetFrameworkVersion');
+      }) || {};
+  // tslint:disable
```

Comment:
What does this do? I haven't found it in the docs :-) Also.. do we need it?

Reply:
left over, it disabled linting. removing this.